### PR TITLE
#119: create View interface and add its implementation in Console class

### PR DESCRIPTION
**Pull request by AVM82**
connect to #119

### CODE REVIEW

File: src/main/java/juja/sqlcmd/view/Console.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package juja.sqlcmd.view;
+
+import java.util.Scanner;
+
+public class Console implements View {
```
Comment (adaleorion) [inline src/main/java/juja/sqlcmd/view/Console.java]: I think this class needs to constructor with input and output streams as parameters

**Review by adaleorion — CHANGES_REQUESTED** [review]
You need to add some tests for your implementation

**Comment by kortov** [comment]
I think tests are needed

File: src/main/java/juja/sqlcmd/view/Console.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Scanner;
+
+public class Console implements View {
+
+    private Scanner scanner;
+    private OutputStream outputStream;
+
+    public Console(OutputStream printStream, InputStream inputStream) {
```
Comment (kortov) [inline src/main/java/juja/sqlcmd/view/Console.java]: Maybe is needed to rename printStream to outputStream?

File: src/test/java/juja/sqlcmd/ConsoleTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    public void tail() {
+        out.reset();
+    }
+
+    @Test
+    public void testWriteSomeText() {
+        String message = "Test Line";
+        view.write(message);
+        String actual = new String(out.toByteArray());
+        assertEquals(message + System.lineSeparator(), actual);
```
Comment (kortov) [inline src/test/java/juja/sqlcmd/ConsoleTest.java]: May be extract System.lineSeparator() to a constant

File: src/test/java/juja/sqlcmd/ConsoleTest.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+    @BeforeClass
+    public static void setup() {
+        out = new ByteArrayOutputStream();
+        view = new Console(System.out, System.in);
+        System.setOut(new PrintStream(out));
+    }
+
+    @After
+    public void tail() {
```
Comment (kortov) [inline src/test/java/juja/sqlcmd/ConsoleTest.java]: tail is a noun "хвост", maybe we should rename this?

File: src/test/java/juja/sqlcmd/ConsoleTest.java [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    @Test
+    public void testWriteWhenEmptyLine() {
+        String message = "";
+        view.write(message);
+        String actual = new String(out.toByteArray());
+        assertEquals(message + System.lineSeparator(), actual);
+    }
+
+    @Test
+    public void testWriteWhenEmptyLine2() {
```
Comment (kortov) [inline src/test/java/juja/sqlcmd/ConsoleTest.java]: Maybe rename method and remove the 2 number?

File: src/main/java/juja/sqlcmd/view/Console.java (hunk 2 above)
Comment (AVM82) [reply src/main/java/juja/sqlcmd/view/Console.java]: Yes of course! This is an artifact after refactoring :) Thanks!

File: src/test/java/juja/sqlcmd/ConsoleTest.java (hunk 5 above)
Comment (AVM82) [reply src/test/java/juja/sqlcmd/ConsoleTest.java]: You're right.

File: src/test/java/juja/sqlcmd/ConsoleTest.java (hunk 4 above)
Comment (AVM82) [reply src/test/java/juja/sqlcmd/ConsoleTest.java]: I will rename this

**Review by adaleorion — APPROVED** [review]

**Pull request merged**